core/store/src/flat_state.rs

```diff
@@ -496,77 +600,15 @@ impl FlatStorageState {
         })))
     }
 
-    /// Get deltas for blocks, ordered from `self.block_hash` to flat state head (backwards chain order).
-    /// If sequence of deltas contains final block, head is moved to this block and all deltas until new head are
-    /// applied to flat state.
-    // TODO (#7327): move updating flat state head to block postprocessing.
-    // TODO (#7327): come up how the flat state head and tail should be positioned.
-    // TODO (#7327): implement garbage collection of old deltas.
-    // TODO (#7327): cache deltas to speed up multiple DB reads.
+    /// Get deltas between blocks `target_block_hash`(inclusive) to flat head(inclusive),
+    /// in backwards chain order. Returns an error if there is no path between these two them.
```

Comment:
I think it would be helpful to say what "backwards chain order" means, it's not clear what "chain order" exactly refers to.